native_client/ctcdecode/mman.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * sys/mman.h
+ * mman-win32
```

Comment:
Is this written by you or coming from some third-party ?
It's to handle `mmap()` ?